pkg/controller/elasticsearch/client/shard.go

```diff
@@ -21,7 +21,7 @@ type ShardLister interface {
 }
 
 func (c *clientV6) ExcludeFromShardAllocation(ctx context.Context, nodes string) error {
-	ctx, cancel := context.WithTimeout(ctx, DefaultReqTimeout)
+	ctx, cancel := context.WithTimeout(ctx, defaultClientTimeout)
```

Comment:
do we need to set the timeout in the context if the http client already has a timeout set internally?